docs/common/istio-ambient-mode.md

```diff
@@ -0,0 +1,353 @@
+
+```console
+$ kubectl get ztunnel -n istio-system
+    NAME      READY   STATUS    VERSION   AGE
+    default   True    Healthy   v1.24.0   16s
+```
+
+### Deploy a sample application
+
+To explore Istio's ambient mode, let's install the sample `Bookinfo application`.
+
+*Steps*
+
+1. Create the `bookinfo` namespace and add a label `istio-discovery=enabled`.
+
+```bash
+$ kubectl create ns bookinfo
+$ kubectl label namespace bookinfo istio-discovery=enabled
+```
+
+2. Deploy the application.
+
+```bash
+$ kubectl apply -n bookinfo -f https://raw.githubusercontent.com/istio/istio/release-1.24/samples/bookinfo/platform/kube/bookinfo.yaml
+$ kubectl apply -n bookinfo -f https://raw.githubusercontent.com/istio/istio/release-1.24/samples/bookinfo/platform/kube/bookinfo-versions.yaml
+```
+
+3. Verify that the application is running.
+
+```bash
+$ kubectl get -n bookinfo pods
+
+    NAME                             READY   STATUS    RESTARTS   AGE
+    details-v1-cf74bb974-nw94k       1/1     Running   0          42s
+    productpage-v1-87d54dd59-wl7qf   1/1     Running   0          42s
+    ratings-v1-7c4bbf97db-rwkw5      1/1     Running   0          42s
+    reviews-v1-5fd6d4f8f8-66j45      1/1     Running   0          42s
+    reviews-v2-6f9b55c5db-6ts96      1/1     Running   0          42s
+    reviews-v3-7d99fd7978-dm6mx      1/1     Running   0          42s
+```
+
+4. Deploy and configure the ingress gateway using the Kubernetes Gateway API.
+
+```bash
+$ kubectl get crd gateways.gateway.networking.k8s.io &> /dev/null || \ 
+{ kubectl apply -f https://github.com/kubernetes-sigs/gateway-api/releases/download/v1.2.0/standard-install.yaml; }
+$ kubectl apply -n bookinfo -f https://raw.githubusercontent.com/istio/istio/release-1.24/samples/bookinfo/gateway-api/bookinfo-gateway.yaml
+```
+    
+Wait for the `bookinfo-gateway` pod running and then get the `productpage` service URL. The wait time depends on your cluster cloud provider. It takes me about one minute from an AWS ELB to be able to access it.
+
+```bash
+$ export INGRESS_HOST=$(kubectl get -n bookinfo gtw bookinfo-gateway -o jsonpath='{.status.addresses[0].value}')
+$ export INGRESS_PORT=$(kubectl get -n bookinfo gtw bookinfo-gateway -o jsonpath='{.spec.listeners[?(@.name=="http")].port}')
+$ export GATEWAY_URL=$INGRESS_HOST:$INGRESS_PORT
+$ echo "http://${GATEWAY_URL}/productpage"
+```
+
+5. Access the application.
+
+Open your browser and navigate to `http://${GATEWAY_URL}/productpage` to view the Bookinfo application.
+If you refresh the page, you should see the display of the book ratings changing as the requests are distributed across the different versions of the reviews service.
+
+6. Add Bookinfo to the Ambient mesh.
+
+```bash
+$ kubectl label namespace bookinfo istio.io/dataplane-mode=ambient
+```
+
+> **Note**:
+> You don't need to restart or redeploy any of the application pods.
+> Unlike the sidecar mode, each pod's container count will remain one in the ambient mode.
```

Comment:
Some pods might have more than one container in their typical deployments, so lets modify the sentence as....

```suggestion
> Unlike the sidecar mode, each pod's container count will remain the same even after adding them to the ambient mesh.
```